middleware/downloadcsv.js

```diff
@@ -1,18 +1,42 @@
 const { parse, transforms: { unwind } } = require("json2csv")
 
 const fields = [
-  "id",
-  "title",
-  "answers.id",
-  "answers.label",
-  "answers.options.label",
-  "answers.options.value"
+  {
+    label: "Author short code",
+    value: (row) => row.alias
+  },
+  {
+    label: "Question",
+    value: (row) => row.title
+  },
+  {
+    label: "Type",
+    value: (row) => row.answers.type
+  },
+  {
+    label: "Answer label",
+    value: (row) => (row.answers.secondaryLabel ? row.answers.label + ' / ' + row.answers.secondaryLabel : row.answers.label )
```

Comment:
```suggestion
    value: (row) => (row.answers.secondaryLabel ? `${row.answers.label} / ${row.answers.secondaryLabel }` : row.answers.label )
```

Small refactoring suggestion; not super important 'though! You can use this syntax, sometimes called an F-String, to do what you had done but in an (often) more readable way.